pkg/cli/public_key_server.go

```diff
@@ -119,7 +119,8 @@ func (c *PublicKeyServerCommand) RunUnstarted(ctx context.Context, args []string
 	keyServer := jvscrypto.NewKeyServer(ctx, kmsClient, c.cfg, h)
 
 	mux := http.NewServeMux()
-	mux.Handle("/healthz", healthcheck.HandleHTTPHealthCheck())
+	// This is the healthz checkpoint.
```

Comment:
nit: seems redundant. same to the other comments. maybe just add the reference link and mention to avoid cloud run reserved paths.